requirements.in

```diff
@@ -1,11 +1,11 @@
-molecule>=4.0.1
-ansible-core>=2.13.2
-ansible-lint>=6.6.0
-kubernetes>=12.0.0
+molecule>=4.0.3
+ansible-core>=2.13.5
+ansible-lint>=6.8.6
+kubernetes>=25.3.0
 molecule-vagrant>=1.0.0
-molecule>=4.0.1
+molecule>=4.0.3
 netaddr>=0.8.0
-pyyaml>=3.11
+pyyaml>=6.0
 yamllint>=1.28.0
 jmespath>=1.0.1
 jsonpatch>=1.32
```

Comment:
We have some duplicates in this file (i.e, molecule and netaddr). So I propose sorting the requirements and deleting the duplicates, like so:

```
ansible-core>=2.13.5
ansible-lint>=6.8.6
jmespath>=1.0.1
jsonpatch>=1.32
kubernetes>=25.3.0
molecule-vagrant>=1.0.0
molecule>=4.0.3
netaddr>=0.8.0
pre-commit>=2.20.0
pyyaml>=6.0
yamllint>=1.28.0
```

Reply:
Good catch! Thank you @sleiner !